Makefile

```diff
@@ -83,26 +83,24 @@ node_modules: package.json package-lock.json
 	  --entrypoint 'npm' \
 	  node ci
 
-upstream/gvisor/bazel-bin/runsc/linux_amd64_pure_stripped/runsc: upstream-gvisor
-	mkdir -p /tmp/gvisor-website/build_output
+bin/generate-syscall-docs: $(GEN_SOURCE)
+	mkdir -p bin/
+	go build -o bin/generate-syscall-docs gvisor.dev/website/cmd/generate-syscall-docs
+
+compatibility-docs: bin/generate-syscall-docs
+	# bazel_user_root is used for caching bazel packages.
+	mkdir -p bazel_user_root/
 	docker run \
 	  --rm \
 	  -v $(PWD)/upstream/gvisor:/workspace \
-	  -v /tmp/gvisor-website/build_output:/tmp/gvisor-website/build_output \
+	  -v $(PWD)/bazel_user_root:/bazel_user_root \
 	  -w /workspace \
 	  --entrypoint 'sh' \
 	  l.gcr.io/google/bazel \
 	  -c '\
 		groupadd --gid $(shell id -g) $(shell id -gn) && \
 		useradd --uid $(shell id -u) --gid $(shell id -g) -ms /bin/bash $(USER) && \
-		su $(USER) -c "bazel --output_user_root=/tmp/gvisor-website/build_output build //runsc"'
-
-bin/generate-syscall-docs: $(GEN_SOURCE)
-	mkdir -p bin/
-	go build -o bin/generate-syscall-docs gvisor.dev/website/cmd/generate-syscall-docs
-
-compatibility-docs: bin/generate-syscall-docs upstream/gvisor/bazel-bin/runsc/linux_amd64_pure_stripped/runsc
-	./upstream/gvisor/bazel-bin/runsc/linux_amd64_pure_stripped/runsc help syscalls -o json | ./bin/generate-syscall-docs -out ./content/docs/user_guide/compatibility/
+		su $(USER) -c "bazel --output_user_root=/bazel_user_root run //runsc -- help syscalls -o json"' | ./bin/generate-syscall-docs -out ./content/docs/user_guide/compatibility/
```

Comment:
Can you use bazel symlinks added to the repo root (bazel-bin, bazel-out, etc) to find files instead of setting `--output_user_root`?

Reply:
Unfortunately no, if the base path is different the symlinks are broken when used from outside the container.